lib/location/location_form_field.dart

```diff
@@ -24,12 +23,6 @@ class LocationFormField extends FormField<TrufiLocation> {
               padding: EdgeInsets.all(4.0),
               child: GestureDetector(
                 onTap: () async {
-                  // Customize search hint
```

Comment:
This part was rather interesting, it sets a "HintText" but the hint text is never read via the getter. Therefore this was deprecated and I removed it instead of repairing it.

Reply:
Yeah, I think we can remove it then. Or is it obvious where the hint should have been displayed? The param would be obsolete as well.

Reply:
I cross-checked the Form Fields that I could see. Nothing really needed that flag. Also, I removed now the parameter, also no obvious behaviour change. Of course, without tests, it is hard to say that there is no Side effect somewhere.